subgraphs/balancer-v2/src/common/metrics.ts

```diff
@@ -218,11 +219,10 @@ export function updateTokenPrice(
   const tokenInfo = calculatePrice(tokenA, tokenAmountIn, weightTokenA, tokenB, tokenAmountOut, weightTokenB);
 
   if (tokenInfo) {
-    let token = Token.load(tokenInfo.address.toHexString());
-    if (!token) token = new Token(tokenInfo.address.toHexString());
+    const token = getOrCreateToken(tokenInfo.address);
     const index = tokenInfo.address == tokenB ? tokenBIndex : tokenAIndex;
     const currentBalance = scaleDown(pool.inputTokenBalances[index], Address.fromString(pool.inputTokens[index]));
-    // We check if current balance multiplied by the price is over 40k USD, if not,
+    // We check if current balance multiplied by the price is over 10k USD, if not,
     // it means that the pool does have too much liquidity, so we fetch the price from
     // external source
     if (currentBalance.times(tokenInfo.price).gt(BigDecimal.fromString("40000"))) {
```

Comment:
It's `10k` in the comments but `40k` in the code

Reply:
nice catch :pray: